packages/thirdweb/src/exports/transaction.ts

```diff
@@ -55,6 +55,7 @@ export {
   toSerializableTransaction,
   type ToSerializableTransactionOptions,
 } from "../transaction/actions/to-serializable-transaction.js";
+export { getSessionTransactions } from "../transaction/transaction-store.js";
```

Comment:
maybe it makes sense to always let people access the store instead here, that way they can still `getValue()` to get the data, but also they can subscribe to updates etc which seems useful?

Reply:
and you could remove the implementation of `getSessionTransactions` entirely (since its essentially a proxy of `getTransactionStore().getValues()` anyways?